src/plugins/workspace/public/components/workspace_picker_content/workspace_picker_content.tsx

```diff
@@ -72,18 +75,16 @@ export const WorkspacePickerContent = ({
     const recentWorkspaces = recentWorkspaceManager.getRecentWorkspaces();
     const updatedList = workspaceList.map((workspace) => {
       const recentWorkspace = recentWorkspaces.find((recent) => recent.id === workspace.id);
-
-      if (recentWorkspace) {
-        return {
-          ...workspace,
-          accessTimeStamp: recentWorkspace.timestamp,
-          accessTime: `viewed ${moment(recentWorkspace.timestamp).fromNow()}`,
-        };
-      }
-      return workspace as UpdatedWorkspaceObject;
+      return {
+        ...workspace,
+        accessTimeStamp: recentWorkspace?.timestamp,
+        accessTime: recentWorkspace
```

Comment:
should we name it as `accessTimeLabel` or `accessTimeDescription` ?

Reply:
Thanks for the review, updated